docs/content/docs/connectors/datastream/kafka.md

```diff
@@ -344,6 +344,16 @@ fetched from Kafka in ```SplitReader```. The state of split, or current progress
 consuming is updated by ```KafkaRecordEmitter``` , which is also responsible for assigning event time
 when the record is emitted downstream.
 
+
+### Triage
```

Comment:
Would this be clearer if it was located in the `Kafka Consumer Metrics` section?

Reply:
That's actually a good point. I take a look.